src/classes/Commands.ts

```diff
@@ -1546,22 +1547,35 @@ export = class Commands {
 
             this.bot.sendMessage(steamID, 'Accepting offer...');
 
+            const partnerId = new SteamID(this.bot.manager.pollData.offerData[offerId].partner);
+            const reply = message.substr(message.toLowerCase().indexOf(offerId) + 11);
+            const adminDetails = this.bot.friends.getFriend(steamID);
+
             this.bot.trades.applyActionToOffer('accept', 'MANUAL', {}, offer).asCallback(err => {
                 if (err) {
                     this.bot.sendMessage(
                         steamID,
                         'Ohh nooooes! Something went wrong while trying to accept the offer: ' + err.message
                     );
+                    return;
+                }
+                // Send message to recipient if includes some messages
+                if (!err && reply) {
+                    this.bot.sendMessage(
+                        partnerId,
+                        'Message from ' + (adminDetails ? adminDetails.player_name : 'admin') + ': ' + reply
+                    );
                 }
             });
         });
     }
 
     private declinetradeCommand(steamID: SteamID, message: string): void {
-        const offerId = CommandParser.removeCommand(message).trim();
+        const parts = message.split(' ');
```

Comment:
Maybe use something else instead of splitting it up into an array. Try with substring and indexOf like you did with the reply